### PR TITLE
Fix the Combo Operator Installation docs

### PR DESCRIPTION
**Pull request by oceanc80**
The [Installing the Combo Operator](https://github.com/operator-framework/rukpak/tree/main/internal/provisioner/plain#installing-the-combo-operator) instructions referenced broken images. This PR updates those references to the correct images.

Closes: https://github.com/operator-framework/rukpak/issues/251

### CODE REVIEW

**Comment by tylerslaton** [comment]
@oceanc80 Could we get this PR rebased for review?

File: internal/provisioner/plain/README.md [hunk 1]
```diff
@@ -135,23 +135,30 @@ resources referenced by the bundle are present on the cluster.
 ### Setup
 
 To experiment with the `plain` provisioner locally, first setup a local cluster, or simply
-have [kind](https://kind.sigs.k8s.io/) installed locally.
+have [kind](https://kind.sigs.k8s.io/) installed and a kind cluster running locally.
```
Comment (tylerslaton) [inline internal/provisioner/plain/README.md:138]: This probably changes once #365 closes since it'll be creating a local copy of the kind binary.

**Review by tylerslaton — COMMENTED** [review]
A few comments. Looks like the CI is failing due to linting issues. Once that gets fixed I'm good with this getting through.

File: internal/provisioner/plain/README.md [hunk 2]
```diff
@@ -166,50 +173,83 @@ spec:
...
+should be present:
+
+```
 $ kubectl -n combo get deployments.apps combo-operator
 NAME             READY   UP-TO-DATE   AVAILABLE   AGE
 combo-operator   1/1     1            1           10s
 
 $ kubectl -n combo get deployments.apps combo-operator -o yaml | grep 'image:' | xargs
-image: quay.io/tflannag/combo:v0.0.1
+image: quay.io/operator-framework/combo-operator:latest
```
[the remark this answers is not in the document]
Comment (oceanc80) [reply internal/provisioner/plain/README.md:233]: At the moment the way we build the combo bundles results in all bundle versions pointing to the latest operator image. Discussion on this in slack [here](https://kubernetes.slack.com/archives/C038B7MF75M/p1652474613798159).

**Review by exdx — COMMENTED** [review]
This looks good, thank you for fixing up the docs! 

My only comment is regarding the commit history -- we generally just rebase off HEAD and incorporate our commits that way, without any merge commits from master onto the branch. This way the git history is clean. Could you cherry pick your commits onto a new up-to-date branch? Since we're just updating that one doc, maybe squashing down to one commit makes sense as well? 

After that this is ready to go in

**Review by timflannagan — COMMENTED** [review]
Can we rebase against the latest main branch commits to make it easier to review the distinct set of changes here?

**Pull request closed without merging**